ert3/evaluator/_evaluator.py

```diff
@@ -54,38 +52,54 @@ def _build_ee_config(evaluation_tmp_dir, ensemble, stages_config, input_records)
         ensemble_size = input_records.ensemble_size
 
     stage_name = ensemble.forward_model.stages[0]
-    step_name = stage_name + "-only_step"
     stage = stages_config.step_from_key(stage_name)
+    commands = stage.transportable_commands
+    command_scripts = [cmd.location for cmd in commands] if commands else []
+    output_locations = [out.location for out in stage.output]
+    jobs = []
+
+    def command_location(name):
+        return next(
+            cmd.location for cmd in stage.transportable_commands if cmd.name == name
+        )
+
+    if stage.function:
+        jobs.append(
+            {
+                "name": stage.function.__name__,
+                "executable": stage.function,
+                "output": output_locations[0],
+            }
+        )
 
-    command_scripts = [cmd.location for cmd in stage.transportable_commands]
-    output_files = [out.location for out in stage.output]
+    for script in stage.script:
+        name, *args = script.split()
```

Comment:
Do we want to do this ourselves? 

I'm just thinking if you want to run this command `ecl_run "C:\My Documents\My First Eclipse Simulation" …` you're screwed.

Anything we can lean on? E.g. something from argparse to get the actual constituents of the command?

Reply:
I share your concern but I am not really changing the behaviour with this PR and I would do any improvements in future improvements PR. 

But I will look if there is anything from argparse that can help.